packages/commonwealth/client/scripts/views/components/KnockNotifications/KnockNotifications.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  KnockFeedProvider,
+  KnockProvider,
+  NotificationFeedPopover,
+  NotificationIconButton,
+} from '@knocklabs/react';
+import React, { useRef, useState } from 'react';
+import app from 'state';
+
+dotenv.config();
```

Comment:
Why are we using dotenv on the client? Does this even work?

Reply:
I assumed that was how the DefinePlugin was working.  I will double check.

This is a good reason why server-only libs shouldn't even be available on the client :-/